Examples/BugReport/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:4.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "BugReport",
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+         .package(url: "https://github.com/mongodb/mongo-swift-driver", from: "0.0.3")
```

Comment:
this makes me realize, we should actually have people do `cat Package.resolved`. Swift rounds up patches to the latest patch version,  but won't automatically install an update when a new one comes out. so without knowing whether they have recently done `swift package update` we can't tell much from what's stated here -- they could still actually be on 0.0.3 or they could be all the way up to 0.0.7. 

I'm not sure if there is a .resolved equivalent for Cocoapods, but if so switch to that as well